scripts/inference.py

```diff
@@ -218,6 +220,24 @@
     default=0,
     help="Set verbosity level (pass flag as `-v`, `-vv`, `-vvv`)"
 )
+parser.add_argument(
+    "--stagger_load",
+    type=int,
+    default=0,
+    help="Stagger model loading to avoid OOM issues on the host"
+)
+parser.add_argument(
+    "--stagger_update_lazyhandle",
+    type=int,
+    default=0,
+    help="Stagger update_lazyhandle to avoid OOM issues on the host"
```

Comment:
can you provide an example of what setting this value does

Reply:
When you launch a 16 way TP model and set `--stagger_update_lazyhandle 2` then it'll only allow up to `2` processes to enter the `update_lazyhandle ` phase of the code at a time instead of all 16.

How about this for a rewording of the help text?
```suggestion
    help="Limit the number of concurrent processes executing the AIU update_lazyhandle phase. Set to 0 to allow all processes"
```